demmt/nvrm_ioctl.h

```diff
@@ -491,18 +491,38 @@ struct nvrm_ioctl_destroy_os_event {
 };
 #define NVRM_IOCTL_DESTROY_OS_EVENT _IOWR(NVRM_IOCTL_MAGIC, NVRM_IOCTL_ESC_BASE+7, struct nvrm_ioctl_destroy_os_event)
 
+struct nvrm_ioctl_status_code {
+	uint32_t domain;
+	uint8_t bus;
+	uint8_t slot;
+	uint32_t status;
```

Comment:
There's 2 byte hole between slot and status ("pahole" is a useful tool if you are unsure about layout). Just in case those bytes have a meaning you should add explicit padding and print it with nvrm_print_pad_*.